helmfile.d/lists/images.yaml

```
@@ -0,0 +1,6 @@
+---
+images:
+  ingressNginxChart:
```

I would probably not include chart in the key as I think it makes sense to group them based on the stack they are part of.
Else we will need to have `opensearchChart.node` and `opensearchDashboardsChart.dashboards` instead of `opensearch.{dashboards,node}`.
```suggestion
  ingressNginx:
```

---

I did this to differentiate between `upstream` and our own `charts`, but you're right, the `stack` level abstraction is better suited for it.

---

Done 8aef89f6